src/Generators/RequestsGenerator.php

```diff
@@ -14,15 +14,11 @@ class RequestsGenerator extends EntityGenerator
     const DELETE_METHOD = 'Delete';
     const GET_METHOD = 'Get';
 
-    public function setRelations($relations)
+    public function setRelationFields(): array
```

Comment:
let's call this new method inside the `setRelations` method to perform this logic only once